examples/config.js

```diff
@@ -0,0 +1,134 @@
+module.exports = {
```

Comment:
We so many copies of this or part of this in the tests. Maybe we should use something to generate all of these automatically instead of relying on copy paste. (When we need to change some default stuff)

Reply:
You may have been referring to something that was in the previous PR, because I don't see many copies here. But yeah, for unit tests, there are 2 configuration files, one used throughout unit tests, the other to test with different configuration values. That second one could be mocked instead. But the purpose is to test the code path with the file.

This file is the only "real" (not test) configuration file

Reply:
You may have been referring to something that was in the previous PR, because I don't see many copies here. But yeah, for unit tests, there are 2 configuration files, one used throughout unit tests, the other to test with different configuration values. That second one could be mocked instead. But the purpose is to test the code path with the file.

This file is the only "real" (not test) configuration file

Reply:
Yes, I meant in general, not necessarly in this PR